### ISSUE TITLE
php 7 codestyle use 

### ISSUE THREAD

**Issue by des1roer** [pahanini/yii2-rest-doc#22]
when use aka php7

    use common/models{
	     Class1, Class2
	}; 

fatal error 

> Parse Error: Syntax error, unexpected '{', expecting T_STRING on line 9
Exception 'yii\base\InvalidParamException' with message 'File /var/www/html/frontend/modules/api/modules/v3/controllers/RecommendationController.php includes more then one class'

**Comment by pahanini** [comment]
Is it legal to have some classes in one file according yii2 standards?

**Comment by des1roer** [comment]
yes. in php 7 u can use multiple uses https://www.tutorialspoint.com/php7/php7_use_statement.htm

**Comment by pahanini** [comment]
https://github.com/phpDocumentor/Reflection/issues/125

**Comment by sdlins** [comment]
@pahanini https://github.com/phpDocumentor/Reflection/issues/125#issuecomment-364738648
